src/NexusMods.App.UI/Controls/Spine/SpineViewModel.cs

```diff
@@ -127,85 +96,42 @@ private Bitmap LoadImageFromStream(Stream iconStream)
     private void NavigateToHome()
     {
         _logger.LogTrace("Home selected");
-        _actions.OnNext(new SpineButtonAction(Type.Home));
-        LeftMenu = _homeLeftMenuViewModel;
+        // TODO:
```

Comment:
This and Downloads will be for another PR?

Reply:
Yup